docs/obis-observer/commands/ReadArchive.md

```diff
@@ -0,0 +1,106 @@
+| `1`  | `byte`                               | Meter id `2`                                                                                                          |
+| `4`  | [Time 2000](../types.md#time-2000)   | Date `2`. The date and time at which the data was captured                                                            |
+| `1`  | [OBIS ID](../types.md#obis-id)       | OBIS ID `1`                                                                                                           |
+| `4`  | `float32`                            | OBIS content `1`                                                                                                      |
+| ...  | ...                                  | ...                                                                                                                   |
+| `1`  | [OBIS ID](../types.md#obis-od)       | OBIS ID `N`                                                                                                           |
+| `4`  | `float32`                            | OBIS content `N`                                                                                                      |
+
+#### Date end flag
+If a byte with the value 0 immediately follows the OBIS content, it indicates the end-of-date flag. In this case, the subsequent 4 bytes will be interpreted as the next date and time.
+
+
+### Examples
+
+| Field              | Value                     | Hex          |
+| ------------------ | ------------------------- | ------------ |
+| command id         | `22`                      | `0x16`       |
+| command size       | `28`                      | `0x1с`       |
+| request id         | `01`                      | `0x09`       |
+| is completed       | `1`                       | `0x01`       |
+| meter id `1`       | `1`                       | `0x01`       |
+| date `1`           | `2023.10.10 20:26:16 GMT` | `0x14560168` |
+| OBIS ID `1`        | `108`                     | `0x6c`       |
+| OBIS content `1`   | `0.20`                    | `0x3e4ccccd` |
+| meter `1` end flag | `0`                       | `0x00`       |
+| meter id `2`       | `2`                       | `0x02`       |
+| date `2`           | `2023.10.10 20:26:15 GMT` | `0x14560167` |
+| OBIS ID `1`        | `8`                       | `0x08`       |
+| OBIS content `1`   | `0.20`                    | `0x3e4ccccd` |
+| OBIS ID `2`        | `108`                     | `0x6c`       |
+| OBIS content `2`   | `0.20`                    | `0x3e4ccccd` |
+
+
+Message hex dump: `16 1c 09 01 01 14 56 01 68 6c 3e 4c cc cd 00 02 14 56 01 67 08 3e 4c cc cd 6c 3e 4c cc cd`
+
+#### error:
+
+If an error occurs, the observer will respond by sending the [Error](./uplink/Error.md) command.
+
+##### Result codes:
+
+| Result code | Description   |
+| ----------- | ------------- |
+| `3`         | Format error. |
+
+## See also
+
+* [Request ID](../types.md#request-id)
+* [Meter ID](../types.md#meter-id)
```

Comment:
```suggestion
```

Reply:
Fixed. Actually, meter ID is used in this command. I made it explicit in the command description.